discovery.md

```diff
@@ -521,14 +545,184 @@ When encoded in JSON, the response format MUST adhere to the following:
 }
 ```
 
-##### `/services?name={name}`
+Note: in the case of `406 Not Acceptable` or `409 Conflict`, it is an
+implementation choice as to how long these responses will be returned. But it
+is expected that a `404 Not Found` will eventually be returned.
+
+#### `POST /services`
+
+This MUST add a Service to the list of Services available at this Discovery
+Endpoint. The Body of the request message MUST contain a valid single Service
+definition, with the exception that it MUST NOT include an `id` attribute.
+The `id` MUST be generated by the Discovery Enpoint.
+
+An error MUST be generated if the request can not be fulfilled.
+
+The follow responses are defined by this specification:
+- `201 Created` if the new Service was added. The response MUST include
+  an HTTP `Location` Header with a value that points to the new Service.
+  The HTTP Response Body MUST include the JSON represenation of the
+  newly created Service.
+- `202 Accepted` to indicate that the request has been accepted but not
+  processed yet. The response MUST include an HTTP `Location` Header with
+  a value that can be used to determine the status of the request. The
+  HTTP Response Body MAY be empty.
```

Comment:
This starts to clarify my questions noted in `GET /services/{id}` and does indicate that the discovery service must be stateful to some degree.